### ISSUE TITLE
quiet flag to suppress progress bar for use in cron

### ISSUE THREAD

**Issue by DoubleAgentDave** [Sude-/lgogdownloader#217]
Hi, I can't see any option for a 'quiet' mode where only errors are printed to stderr, is there any chance one could be added? The problem I'm facing is that when the program is run as a cron job then the system journal is spammed with updates to the progress bar. An extra flag with '-cron' or '-quiet' to suppress the progress bar would solve this.

Thanks!

**Comment by endrift** [comment]
I'd like to echo this request. I run this weekly via cron and the output from it is, well, excessive. Even just detecting it's not running on a tty and muting the progress bar would be a huge improvement considering I get emails every week that are way bigger than they need to be as a result.

**Comment by DoubleAgentDave** [comment]
The log files the program generates are great, no progress bars or anything. But when you run something in cron all the stdout/stderr stuff goes straight into the system journal, which makes scrolling through all the lgogdownloader scroll bar stuff and output really annoying.

**Comment by shakeyourbunny** [comment]
Yes, it would be REALLY great that if the logfile (via stdout) is not excessive and nearly unparsable, even for the human eye.

I'd suggest some sort of parameters, eg:

--logfile filename (default ist stdout or if you put an '-' as the logfile name)

Also, perhaps for better (automated) consumption, it would be neat, if the logfile is just a plain CSV file with fields for timestamp, result, game name, game id, filename, file size, file time. It also should be written once a file has been downloaded in ideal circumstances.

If something non game-specific goes wrong, game name should contain "lgogdownloader / http(curl) / gog" or something, alongside for the result code in game id and then as game name the message

Would really be neat ;D

**Comment by Missingmew** [comment]
Just set up lgogdownloader as a systemd service and would like to see this as a feature. The alternative is sending stdout to /dev/null which isnt a great choice either.
